Wireframe/index.html

```diff
@@ -3,31 +3,57 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Elizabeth Tabala" />
     <title>Wireframe</title>
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <header>
-      <h1>Wireframe</h1>
-      <p>
-        This is the default, provided code and no changes have been made yet.
-      </p>
-    </header>
-    <main>
-      <article>
-        <img src="placeholder.svg" alt="" />
-        <h2>Title</h2>
-        <p>
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam,
-          voluptates. Quisquam, voluptates.
-        </p>
-        <a href="">Read more</a>
-      </article>
-    </main>
-    <footer>
-      <p>
-        This is the default, provided code and no changes have been made yet.
-      </p>
-    </footer>
+    <div class="start">
+      <header>
+            <h1>Wireframe</h1>
+            <p>
+              What do we need to know about it?
+            </p>
+          </header>
+          <main>
+            <article>
+              <img src="14.png" alt="an example of a README file" />
+              <h2>README file</h2>
+              <p>
+                A README file, or "Read Me" file, is a document that provides essential information about a project, program, or dataset. 
```

Comment:
If you have two paragraphs, please, wrap them into separate `p` tags (p stands for `paragraph`).